package.json

```diff
@@ -20,15 +20,16 @@
     "@redhat-cloud-services/frontend-components": "^5.2.0",
     "@redhat-cloud-services/frontend-components-notifications": "^4.1.0",
     "@redhat-cloud-services/frontend-components-utilities": "^5.0.13",
+    "@redhat-cloud-services/host-inventory-client": "^4.0.1",
     "@scalprum/react-core": "^0.1.1",
-    "@redhat-cloud-services/host-inventory-client": "^4.1.0",
     "@redhat-cloud-services/javascript-clients-shared": "^2.0.0",
     "@redhat-cloud-services/remediations-client": "^3.1.0",
     "@redhat-cloud-services/sources-client": "^3.0.1",
+    "@scalprum/react-core": "^0.1.1",
```

Comment:
Questionable, potentially not good. This is very very old. 

What prompted adding this?

Reply:
oh, I see. it was already there, but just moved... weird. that this old version is here.

Reply:
OK. quick search, it's not imported anywhere, so we can probably just remove it.

Reply:
I honestly didnt remember doing that at all, I can chop that thang outa there